.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,3 +35,21 @@ or from https://gitmoji.dev/
 ## How to test
 
 <!-- Give REVIEWERS some hits or code snippets on how could this be tested -->
+
+## DevOps Checklist
+<!--
+
+Some checks that might help your code run stable on production, and help devops assess criticality.
+
+Modified from https://oschvr.com/posts/what-id-like-as-sre/
+
+
```

Comment:
This is a good start. After some iterations I would like to come up if possible with some more concrete checkpoints